ndvi_forecast/1_download_data.ipynb

```diff
@@ -224,7 +226,7 @@
    "source": [
     "###  Submit Satellite Jobs\n",
     "Create a satellite job for a given set of boundaries using Azure Farmbeats satellite_data_ingestion_job and SatellitDataIngestionJob() methods. \n",
-    "This returns a pollable object for each satellite job. We can query this object to know the status of each job until gets completed. Once the job succeeded, all satellite scenes will be downloaded in PaaS for given duration and location of intereset. "
+    "This returns a pollable object for each satellite job. We can query this object to know the status of each job until gets completed. Once the job succeeds, all satellite scenes are downloaded in FarmBeats for given duration and location of intereset. "
```

Comment:
1) Interest - typo
2) Do you want to call it location of interest or area of interest?

Something like 

...all satellite scenes are downloaded into FarmBeats datastore for the given duration and area of interest.

Reply:
We are expecting closed boundary (polygon) geo-coordinates which represent an area less than 10000 hectares. I will check with Debasis tomorrow if we should change to location of interest. If we can change it, i will do it as part on next PR.